src/tests/execute/execute_remove_as_seller.rs

```diff
@@ -1,253 +0,0 @@
-#[cfg(test)]
-mod execute_remove_as_seller_tests {
```

Comment:
I'm forgetting why we are dropping this, we execute this when a seller rejects a bwic, is there just no need to mutate the contract state for this?

Reply:
Oh, looks like we aren't even doing that in V2...